server/master/src/main/java/org/apache/accumulo/master/recovery/RecoveryManager.java

```diff
@@ -71,7 +71,7 @@ public RecoveryManager(Master master) {
       List<String> workIDs = new DistributedWorkQueue(ZooUtil.getRoot(master.getInstance()) + Constants.ZRECOVERY, aconf).getWorkQueued();
       sortsQueued.addAll(workIDs);
     } catch (Exception e) {
-      log.warn("{}", e.getMessage(), e);
+      log.warn(e.getMessage(), e);
```

Comment:
The old version is a little bit better, I think, because it prevents the exception's message from being interpreted as a format string. In other words, if the message itself contains braces, then this won't behave as expected. This is unlikely (because braces in the exception message are rare), but possible. This change occurs in a few other places, as well.